ui/app/AppLayouts/Chat/panels/communities/ChatPermissionQualificationPanel.qml

```diff
@@ -0,0 +1,123 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import utils 1.0
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import AppLayouts.Chat.popups.community 1.0
+import AppLayouts.Chat.views.communities 1.0
+
+Item {
+    id: root
+
+    property string channelName
+    property var assetsModel
+    property var collectiblesModel
+    property var viewOnlyHoldingsModel
+    property var moderateHoldingsModel
+    property var viewAndPostHoldingsModel
+
+    QtObject {
+        id: d
+        property int panelRowSpacing: 4 // by design
+    }
+
+    RowLayout {
+        anchors.fill: parent
+        spacing: d.panelRowSpacing
+        StatusBaseText {
+            text: qsTr("To post, hold")
+            font.pixelSize: Style.current.primaryTextFontSize
+            color: Style.current.secondaryText
+        }
+
+        StatusScrollView {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 20
+            contentWidth: tokenRow.implicitWidth
+            padding: 0
+            Row {
+                id: tokenRow
+                Repeater {
+                    id: repeater
+                    model: root.viewOnlyHoldingsModel
```

Comment:
I'm thinking that maybe the correct model here is `viewAndPostHoldingsModel` since here we are disabling the chat input, so we are disabling the ability of posting due to no requirements met to post. 

In case of `viewOnlyHoldingsModel`, I'm not sure about the design / flow. Maybe requirements need to be displayed also in the placeholder while the chat messages are blured.. However, we need to ask the design team for clarification here. 

Does it make sense?

Reply:
> I'm thinking that maybe the correct model here is viewAndPostHoldingsModel since here we are disabling the chat input,

But they can still view right? at least this is what I get from figma, they can view but not post

Reply:
Yes, they can view but not post, so the model that prevents to post is the `viewAndPost` one. Maybe the naming is confusing.. and should be only `post`? Bc I don't think it makes sense to have a permission where you can post but not view.. :thinking:

Reply:
> Bc I don't think it makes sense to have a permission where you can post but not view..

Just as an FYI: if there's a `ViewAndPost` permission that you satisfy, then you automatically overrule the `ViewOnly` permission.

So you'd never end up in the scenario you've described.